src/config.go

```diff
@@ -1,41 +1,39 @@
 package opennox
 
 import (
+	"github.com/adrg/xdg"
+	"github.com/spf13/viper"
 	"os"
 	"path/filepath"
-	"runtime"
-
-	"github.com/spf13/viper"
 
 	"github.com/noxworld-dev/opennox-lib/env"
 	"github.com/noxworld-dev/opennox-lib/log"
 )
 
 func init() {
-	wd, err := os.Getwd()
-	if err != nil {
+	viper.SetConfigName(configName)
+	viper.SetConfigType(configExt)
+
+	// Snap
+	if snapCommonDir := env.AppUserDir(); snapCommonDir != "" {
+		viper.AddConfigPath(snapCommonDir)
+	}
+
+	// Current working directory unless it's in a location only for binaries
+	wd, err1 := os.Getwd()
+	if err1 != nil {
```

Comment:
```suggestion
	wd, err := os.Getwd()
	if err != nil {
```
No need to set a special name for it.